travis.sh

```diff
@@ -64,7 +64,9 @@ echo "Testing branch $TRAVIS_BRANCH of $DOWNSTREAM_REPO_NAME"
 # Set apt repo
 sudo -E sh -c 'echo "deb $ROS_REPOSITORY_PATH `lsb_release -cs` main" > /etc/apt/sources.list.d/ros-latest.list'
 # Common ROS install preparation
-wget http://packages.ros.org/ros.key -O - | sudo apt-key add -
+# apt key acquisition. Since keyserver may often become accessible, backup method is added.
+if [ ! "$APTKEY_STORE_SKS" ]; then export APTKEY_STORE_SKS="hkp://ha.pool.sks-keyservers.net"; fi  # Export a variable for SKS URL for break-testing purpose.
+sudo apt-key adv --keyserver $APTKEY_STORE_SKS --recv-key 0xB01FA116 || ((echo 'Fetching apt key from SKS keyserver somehow failed. Trying to get one from alternative.\n'; wget https://raw.githubusercontent.com/ros/rosdistro/master/ros.key -O - | sudo apt-key add -) || (echo 'Fetching apt key by an alternative method failed too. Exitting since ROS cannot be installed.'; error))
```

Comment:
The only reason why I exported SKS server URL as a variable is that I wanted to add a job to test the situation with network being unavailable, and I implemented that by passing wrong SKS server URL (that's what I'm doing in [this](https://github.com/ros-industrial/industrial_ci/pull/42/files/dbc496b1f2c3d1ee57d4a7bcc0ff059e09f35f1f#diff-354f30a63fb0907d4ad57269548329e3R20) and the following lines). That being said I have few reason to export `githubusercontent.com`. Does this make sense?